tests/test_data.py

```diff
@@ -3156,3 +3156,21 @@ def test_export_bldg_threshold(self):
         prj.buildings[-1].thermal_zones[-1].use_conditions.with_ideal_thresholds = True
         prj.calc_all_buildings()
         prj.export_aixlib()
+
+    def test_tz_naming(self):
+
+        from teaser.logic.buildingobjects.building import Building
+        from teaser.logic.buildingobjects.thermalzone import ThermalZone
+
+        # warnings for not calculated buildings
+        bld = Building(parent=prj)
+        tz1 = ThermalZone(parent=bld)
+        tz1.name = "living"
+
+        tz2 = ThermalZone(parent=bld)
+        tz2.name = "kitchen"
+
+        tz3 = ThermalZone(parent=bld)
+        tz3.name = "living"
+
+        assert tz3.name == "living_1"
```

Comment:
Not important, but maybe assert all three zones?
Who knows what might go wrong in future adaptions.

tz1.name == "living" 
tz2.name == "kitchen"

Reply:
added by f134f3f79a75b2eaefcd6263bc1deabf5aceeec7